librz/bin/format/luac/luac.c

```diff
@@ -0,0 +1,69 @@
+//
```

Comment:
Would it be possible to use SPDX style? https://reuse.software/faq/#quick-summary

Reply:
Yeah, it should follow the project header format.

Reply:
Ok, I will apply the format in these days. Btw, is there any other information about rizin development besides `DEVELOPER.md` and `CONTRIBUTING.md` ?

Reply:
Not yet, sorry :( We are going to work on it soonish though! (see https://github.com/rizinorg/rizin/issues/736 and https://github.com/rizinorg/rizin/issues/286). Don't hesitate to ask though, and if you want you can join us on Mattermost to discuss things https://rizin.re/community/